extension/icu/include/icu-datefunc.hpp

```diff
@@ -58,6 +58,16 @@ struct ICUDateFunc {
 	static int32_t ExtractField(icu::Calendar *calendar, UCalendarDateFields field);
 	//! Subtracts the field of the given date from the calendar
 	static int64_t SubtractField(icu::Calendar *calendar, UCalendarDateFields field, timestamp_t end_date);
+	//! Adds the timestamp and the interval using the calendar
+	static timestamp_t Add(icu::Calendar *calendar, timestamp_t timestamp, interval_t interval);
+	//! Adds the interval and the timestamp using the calendar
+	static timestamp_t Add(icu::Calendar *calendar, interval_t interval, timestamp_t timestamp);
```

Comment:
Isn't this just the previous function with the arguments reversed?

Reply:
`timestamp_t Add(icu::Calendar *calendar, interval_t interval, timestamp_t timestamp)` has been removed.